docs/question/index.md

```diff
@@ -1,8 +1,16 @@
 # Questions
 
-**NOTE:** Any time you edit a question `info.json` file on a local copy of PrairieLearn, you need to click “Load from disk” to reload the changes. Edits to HTML or Python files can be picked up by reloading the page. You might need to generate a new variant of a question to run new Python code.
+!!! note "Legacy questions"
 
-**NOTE:** New-style PrairieLearn questions are marked with `"type": "v3"`. This documentation only describes new-style questions, although old-style v2 questions are still supported in the code.
+    Modern PrairieLearn questions are marked with `"type": "v3"`. This documentation describes modern v3 questions, although [legacy v2 questions](../oldQuestion.md#question-types-for-v1v2-format) are still supported by PrairieLearn, though will eventually be phased out.
```

Comment:
Nit: realistically, we'll never remove support from this.

I'd be inclined to remove this note entirely. Folks who are using v2 question know what they're doing, and any new user from the past 7-8 years shouldn't need to know or care anything about v2 questions.

Reply:
8de961fa8bacdf280e06dd8f5e73d124836d5b15